src/multiplayer-servers/getting-started/setup-your-environment.md

```diff
@@ -0,0 +1,57 @@
+# Setup your Environment
+
+In this section you will learn how to setup an Environment and a Region in that Environment.
+
+## Prerequisites
+
+In order to follow this guide, make sure you have the following:
+
+* User credentials to access your GameFabric UI with permissions to create Environments and Regions
+
+Log into the GameFabric UI before proceeding.
+
+## Create an Environment
+
+Environments are a mechanism for isolating groups of resources. They can therefore be used to separate production, staging, testing, 
+and any other environments and ensure that they don't interfere with each other.
+
+First, visit the "Environments" section of the UI by clicking the pencil next to Environments in the menu.
```

Comment:
```suggestion
First, visit the "Environments" section of the UI by clicking the pencil next to Environments in the sidebar menu.
```

Maybe clearer which menu we talk about this way? Since they might be on a page which already has another menu and they might be dumb